source/glest_game/menu/menu_state_custom_game.cpp

```diff
@@ -4441,7 +4437,7 @@ namespace Glest
           } while (++i < GameConstants::maxPlayers);
         }
 
-        updateNetworkSlots ();
+        // updateNetworkSlots ();
```

Comment:
I added this in my original patch. If player 7 was in a slot on an 8 player map, and then the map was switched to a 4 player map, the player in slot 7 would be invisible. Using updateNetworkSlots() seemed to fix that. However, I've noticed a longer than usual delay when hitting "play game" and some other unusual behavior (not worth mentioning yet). So I'm commenting this out for now, and plan to work out a few of the other bugs first.